### PR TITLE
Popup changes

### PR DESCRIPTION
**Pull request by Quadriyanney**
1. Changed how error is displayed if a wrong passphrase is entered in AddAccount screen.

Before: Toast message

After: TextView error

2.Changed how error is displayed if text-field is left empty on the screen to 'Enter Current Password' when a user is about to change spending or startup password.

Before: Snackbar message

After: TextView error

### CODE REVIEW

**Comment by Quadriyanney** [comment]
whatsapp image 2019-02-21 at 10 36 48

Improved on the look and position of the toast displayed when a text is copied to clipboard.

**Comment by Quadriyanney** [comment]
1. Changed how errors are displayed if text-fields in AddAccount screen are left empty when button is clicked.

Before: Toast message
screenshot_20190223-010029_decred wallet testnet
screenshot_20190223-010037_decred wallet testnet

After: TextView error
screenshot_20190222-104400_decred wallet testnet 
screenshot_20190222-104412_decred wallet testnet

File: app/src/main/res/layout/info_dialog.xml [hunk 1]
```diff
@@ -25,10 +25,11 @@
         android:paddingBottom="10dp">
 
         <androidx.appcompat.widget.AppCompatImageView
+            android:layout_marginTop="8dp"
             android:id="@+id/icon"
             android:layout_width="60dp"
             android:layout_height="60dp"
-            android:visibility="gone" />
+            android:visibility="visible" />
```
Comment (C-ollins) [inline app/src/main/res/layout/info_dialog.xml]: Revert this to `gone`. Image is only shown if provided

File: app/src/main/res/values/strings.xml [hunk 2]
```diff
@@ -549,6 +549,6 @@
     <string name="no_wifi">No Wi-Fi</string>
     <string name="wifi_sync_notice">Would you like to sync anyways?</string>
     <string name="not_connected_to_wifi">Not connected to WiFi.</string>
-    <!--/WIFI Sync-->
```
Comment (C-ollins) [inline app/src/main/res/values/strings.xml]: Add closing tag for Wifi Sync string category

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -530,6 +529,7 @@
     <string name="required_to_enter_app">Required to enter app.</string>
     <string name="changing_pin">Changing PIN…</string>
     <string name="security_page_wallet_sync">Please wait for your wallet to finish synchronizing.</string>
+    <string name="field_cannot_be_empty">Field cannot be empty</string>
```
Comment (C-ollins) [inline app/src/main/res/values/strings.xml]: Move this string to the right category

File: app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -110,7 +110,6 @@
     <string name="license">License</string>
     <string name="key_license" translatable="false">license</string>
     <string name="always">Always</string>
-    <string name="not_synced">Not Synced</string>
```
Comment (C-ollins) [inline app/src/main/res/values/strings.xml]: not_synced is best placed in this category

File: settings.gradle [hunk 5]
```diff
@@ -1,7 +1,6 @@
+include ':app', ':dcrlibwallet'
```
Comment (C-ollins) [inline settings.gradle]: Revert the changes made to this file. dcrlibwallet is already included in build.gradle as a library and should not be compiled as a project.

File: app/src/main/java/com/dcrandroid/util/Utils.java [hunk 6]
```diff
@@ -349,8 +349,8 @@ public static void copyToClipboard(Context ctx, String copyText, String successM
         TextView tv = vi.findViewById(android.R.id.message);
         tv.setText(successMessage);
         Toast toast = new Toast(ctx);
-        toast.setGravity(Gravity.TOP | Gravity.CENTER, 0, 50);
-        toast.setDuration(Toast.LENGTH_SHORT);
+        toast.setGravity(Gravity.BOTTOM | Gravity.CENTER, 0, 50);
+        toast.setDuration(Toast.LENGTH_LONG);
```
Comment (C-ollins) [inline app/src/main/java/com/dcrandroid/util/Utils.java]: I prefer that the caller should decided the gravity and duration of the Toast. It not necessary to stay long in all cases especially when the Toast text is short.
